js/modules/types/attachment.js

```diff
@@ -163,3 +164,5 @@ exports.deleteData = (deleteAttachmentData) => {
     await deleteAttachmentData(attachment.path);
   };
 };
+
+exports.save = AttachmentTS.save;
```

Comment:
Quick workaround to avoid exposing `AttachementTS`. We can merge these more formally when the time comes.